platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/service/command/AbstractProfileCommand.java

```diff
@@ -0,0 +1,202 @@
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Extends the {@link AbstractApplicationCommand} with options and styling for profile subcommands
+ */
+public abstract class AbstractProfileCommand extends AbstractApplicationCommand {
+
+    @Option(name = "--dev", description = "Show development profiles", required = false)
+    boolean showDevProfiles = false;
+
+    Path developmentProfilePath = Paths.get(System.getProperty("ddf.home"), "etc", "admin", "profiles");
+
+    protected static final String DEV_PROFILE_INSTALL_FEATURES = "install-features";
+    protected static final String DEV_PROFILE_UNINSTALL_FEATURES = "uninstall-features";
+    protected static final String DEV_PROFILE_START_APPS = "start-apps";
+    protected static final String DEV_PROFILE_STOP_BUNDLES = "stop-bundles";
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(AbstractProfileCommand.class);
+
+    /**
+     * Check if an installation profile exists
+     * @param installProfiles the list of installation profiles
+     * @param profileName the name of the profile to check for
+     * @return {@link Boolean} profile exists?
+     */
+    protected boolean profileExists(List<Feature> installProfiles, String profileName) {
+        return installProfiles.stream().anyMatch(feature -> feature.getName().contains(profileName));
+    }
+
+    /**
+     * Check if a dev profile exists (Experimental)
+     * @param profileName name of profile to check for
+     * @return {@link Boolean} profile exists?
+     */
+    protected boolean devProfileExists(String profileName) {
+        return Paths.get(developmentProfilePath.toString(), profileName + ".json").toFile().exists();
+    }
+
+    /**
+     * Get the {@link Feature} that makes up a profile
+     * @param applicationService {@link ApplicationService}
+     * @param profileName Name of profile to get
+     * @return {@link Feature}
+     */
+    protected Feature getProfile(ApplicationService applicationService, String profileName) {
+        return applicationService.getInstallationProfiles().stream()
+                .filter(i -> i.getName().contains(profileName))
+                .findFirst()
+                .get();
+    }
+
+    /**
+     * Gets the content of a dev profile
+     * @param profileName Name of profile to get
+     * @return {@link Map}
+     */
+    protected Map<String, List<String>> getDevProfile(String profileName) {
+        FileInputStream fileStream;
+        File devProfileFile = Paths.get(developmentProfilePath.toString(), profileName + ".json").toFile();
+        if (!isValidJsonFile(devProfileFile)) {
+            return null;
+        }
+
+        try {
+            fileStream = new FileInputStream(devProfileFile);
+        } catch (FileNotFoundException e) {
+            LOGGER.debug("File not found.");
+            return null;
+        }
+
+        try {
+            return (Map<String, List<String>>) fromJson(IOUtils.toString(fileStream));
```

Comment:
❗️ Should not be calling the deprecated version of this file. Should specify the character set (UTF-8).